niceman/resource/tests/test_docker_container.py

```diff
@@ -140,7 +148,27 @@ def test_dockercontainer_class():
         ]
         client.assert_has_calls(calls, any_order=True)
 
-
-
-
-
+@with_tempfile(content="abc 123")
+def test_docker_session(script=None):
+    config = {
+        'name': 'test-docker-container',
+        'type': 'docker-container',
+        'container_name': 'testing-container'
+    }
+    resource = ResourceManager.factory(config)
+    resource._container == None
+    session = resource.get_session()
+    assert resource._container['Command'] == '/usr/sbin/sshd -D'
+
+    assert session.exists(script) == False
+    session.put(script, "/", owner=None, group=None)
+    assert session.exists(script) == True
```

Comment:
would be useful to verify correct operation of providing owner and group settings here, as noted above there is a bit of inconsistencies ATM